deployment/openshift/about.md

```diff
@@ -1,12 +1,166 @@
+
+```console
+$ sudo dnf install origin-clients docker
+```
+
+Now, let's start our local all-in-one cluster. The command `oc cluster up` takes care of everything:
+
+```console
+$ oc cluster up
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+-- Checking for openshift/origin:v1.3.1 image ...
+   Pulling image openshift/origin:v1.3.1
+   Pulled 0/3 layers, 3% complete
+   Pulled 1/3 layers, 71% complete
+   Pulled 2/3 layers, 86% complete
+   Pulled 3/3 layers, 100% complete
+   Extracting
+   Image pull complete
+-- Checking Docker daemon configuration ... FAIL
+   Error: did not detect an --insecure-registry argument on the Docker daemon
+   Solution:
+
+     Ensure that the Docker daemon is running with the following argument:
+        --insecure-registry 172.30.0.0/16
+```
+
+If you are running a default installation of Docker, you will run into the error above.
+That is okay. We will follow the suggestion on screen. The good news is that `oc cluster up` downloaded the Docker image for running OpenShift Origin, so next time we run it, we already have the images offline.
+
+To change the Docker configuration permanently, edit (as the `root` user) the file `/etc/sysconfig/docker` and add/uncomment the following line:
+
+```
+INSECURE_REGISTRY='--insecure-registry 172.30.0.0/16'
+```
+
+Then, restart the Docker daemon:
+
+```console
+$ sudo systemctl restart docker
+```
+
+And let's run `oc cluster up` again:
+
+```console
+$ oc cluster up
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+-- Checking for openshift/origin:v1.3.1 image ... OK
+-- Checking Docker daemon configuration ... OK
+-- Checking for available ports ... OK
+-- Checking type of volume mount ...
+   Using nsenter mounter for OpenShift volumes
+-- Creating host directories ... OK
+-- Finding server IP ...
+   Using 10.0.2.15 as the server IP
+-- Starting OpenShift container ...
+   Creating initial OpenShift configuration
+   Starting OpenShift using container 'origin'
+   Waiting for API server to start listening
+   OpenShift server started
+-- Installing registry ... OK
+-- Installing router ... OK
+-- Importing image streams ... OK
+-- Importing templates ... OK
+-- Login to server ... OK
+-- Creating initial project "myproject" ... OK
+-- Server Information ...
+   OpenShift server started.
+   The server is accessible via web console at:
+       https://10.0.2.15:8443
+
+   You are logged in as:
+       User:     developer
+       Password: developer
+
+   To login as administrator:
+       oc login -u system:admin
+
+```
+
+Done! Follow the instructions on the screen to access the OpenShift Origin Web Console.
+
+## Deploying your first application
```

Comment:
Good idea, though I feel it would require a bit more context in the new page, because the current content assumes you have just run `oc cluster up`, and therefore are logged in and working in a project.

Maybe we can do that when/if we expand the content with more specific parts? There is room, for example, to writing more about the dev preview, how to login, other common operations, etc.